src/DialogTitle/DialogTitle.tsx

```diff
@@ -29,7 +29,7 @@ const DialogTitle = ({ children, onClose, ...other }: DialogTitleProps) => {
       {onClose ? (
         <Box
           sx={{
-            minWidth: 60
+            textAlign: "right"
```

Comment:
I'm not sure you need this textAlign. I don't think it's doing anything. I think maybe even the whole Box can be removed here. I tested locally and removing the box completely seems fine but please check on your side too as we saw you had diffierent behaviour to me yesterday.